src/Settings.pm

```diff
@@ -189,6 +191,7 @@ sub parseArguments {
 		'mon_control=s',      \$mon_control_file,
 		'items_control=s',    \$items_control_file,
 		'shop=s',             \$shop_file,
+		'buyer_shop=s',		  \$buyer_shop_file,
```

Comment:
don't forget to put a default `buy_shop.txt` file on control folder

Reply:
its already there
```
https://github.com/OpenKore/openkore/pull/2119/files#diff-c02ec3dcf568b2ade692634056827dacR1
```